MODULE.bazel

```diff
@@ -1,8 +1,9 @@
 module(
     name = "pcre2",
-    version = "10.40",
+    version = "10.45-DEV",
```

Comment:
note that in this repo the version here doesn't really matter, most repos just set them to `0` and then when submitting to the bazel-central-registry patch in the correct version number, that way you don't have to worry about updating this as part of the release process

Reply:
OK, good to know. I don't mind setting it to the actual version number however.